src/tito/cli.py

```diff
@@ -630,6 +630,9 @@ def __init__(self):
                 default=False,
                 help=("Don't automatically create a changelog "
                     "entry for this tag if none is found"))
+        self.parser.add_option("--no-default-changelog", action="store_true",
+                default=False,
+                help=("Don't create the changelog entries but write the header"))
```

Comment:
I'm a bit out of touch with my own code, how does this differ from --no-auto-changelog exactly?

Reply:
Ping @mcalmer 

Isn't this related to the fact that we only add headers to the .changes file, but nothing else since we maintain the changelog entries manually and not from the git commits?

Reply:
If that's the case, then maybe this change isn't necessary, because `--no-auto-changelog` would be sufficient. You're manually managing the changes file entirely at that point...

Reply:
but does --no-auto-changelog write the header?

Reply:
No. It writes no entry at all. But why not just invoke `osc vc` and go that route?